### PR TITLE
[2.1.1.x] adjustments for dhc

### PR DESCRIPTION
**Pull request by Nokius**

### CODE REVIEW

File: sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa [hunk 1]
```diff
@@ -1,107 +0,0 @@
...
-load-module module-null-sink sink_name=sink.null rate=48000
-
-load-module module-droid-card rate=48000 mute_routing_before=24576 mute_routing_after=4096
-load-module module-droid-glue
-
-load-module module-null-sink sink_name=sink.fake.sco rate=8000 channels=1
-load-module module-null-source source_name=source.fake.sco rate=8000 channels=1
-load-module module-bluez4-discover sco_sink=sink.fake.sco sco_source=source.fake.sco
-
-load-module module-policy-enforcement route_sources_first=true
```
Comment (saidinesh5) [inline sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: We had to add this file for the route_sources_first=true to get the radio working. So we need this file. If there are any updates to the file provided by upstream, we could probably merge them into ours?

Comment (Nokius) [reply sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: Thanks mate you are right!

Comment (Nokius) [reply sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: holed the horses

Comment (Nokius) [reply sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: let me test all :) 

May I was to early with PR's

I know I sad last night I have no time but I was somehow interested to do it  😄

Comment (saidinesh5) [reply sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: Heh that's why pull requests are nice..  things like these get caught easily..

Comment (Nokius) [reply sparse/etc/pulse/arm_qualcomm_msm_8974_cancro_flattened_device_tree_000b.pa]: no i removed it on purposes but I missed the fmradio my bad

**Pull request closed without merging**